data/projects/improve_diff_handling.md

```diff
@@ -1,24 +1,44 @@
-Bears should be able to offer more than just one possible patch
-for an issue. Users should be able to choose between different patches, e.g.
-for different possible spelling corrections.
-
-In addition bears should be able to offer templated patches, e.g. to offer
+One advantage of using coala is that some Bears offer automatically generated
+diffs to fix the problems they detect. This mechanism works smoothly as of now
+and even applies the proposed diff automatically to the problematic file if the
+user so desires. This is very helpful to the users. At the same time every
+issue in code can't have a single correct solution. The fix can and should
```

Comment:
I'D reword it to "At the same time, not every issue in the code can have just a single correct solution"